applications/PfemFluidDynamicsApplication/custom_python/add_custom_strategies_to_python.cpp

```diff
@@ -63,35 +64,38 @@ namespace Kratos
             typedef NodalTwoStepVPStrategy<SparseSpaceType, LocalSpaceType, LinearSolverType> NodalTwoStepVPStrategyType;
             typedef NodalTwoStepVPStrategyForFSI<SparseSpaceType, LocalSpaceType, LinearSolverType> NodalTwoStepVPStrategyForFSIType;
             typedef GaussSeidelLinearStrategy<SparseSpaceType, LocalSpaceType, LinearSolverType> GaussSeidelLinearStrategyType;
+
+            // Clustom Settings Types
+            typedef TwoStepVPSolverSettings<SparseSpaceType, LocalSpaceType, LinearSolverType> TwoStepVPSolverSettingsType;
             //********************************************************************
             //*************************SHCHEME CLASSES****************************
             //********************************************************************
 
             py::class_<VPStrategyType, VPStrategyType::Pointer, BaseSolvingStrategyType>(m, "VPStrategy")
-                .def(py::init<ModelPart &, LinearSolverType::Pointer, LinearSolverType::Pointer, bool, unsigned int>())
+                .def(py::init<ModelPart &, TwoStepVPSolverSettingsType &>())
```

Comment:
@AFranci After speaking with ruben, I think that maybe this is not correct but rather the two steps should derive from ImplicitSolvingStrategy and use `ImplicitBaseSolvingStrategyType` as a base rather than use `SolvingStrategy` and `BaseSolvingStrategyType`